.github/workflows/be-dev-deploy-closed-prs.yml

```
@@ -1,11 +1,11 @@
 name: BE Dev Deploy Closed Prs
 
 on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)